packages/plugin/src/rules/require-description.ts

```diff
@@ -147,6 +162,10 @@ const rule: GraphQLESLintRule<[RequireDescriptionRuleConfig]> = {
       }
     }
 
+    if (operationFieldDefinition) {
+      kinds.add(':matches(ObjectTypeDefinition, ObjectTypeExtension)[name.value=/Query|Mutation|Subscription/] > FieldDefinition');
```

Comment:
this will not work for root types that are named differently.
example:
```graphql
type MyQuery {
  user: User
}

schema {
  query: MyQuery
}

Reply:
example of fixing https://github.com/B2o5T/graphql-eslint/blob/3e8f71f7ef60dd6e8d5a4605e7ca0bdd2cec2f86/packages/plugin/src/rules/no-root-type.ts#L76

Reply:
I add these in, but they're hardcoded to avoid requiring the schema for the whole rule. If you'd rather these not be hardcoded and to match `no-root-type`'s dynamic fetching of the names, I'd be happy to convert this rule so that the schema is exposed to it.